infra/prod/_modules/app_services/app_service_fims_oidc_provider.tf

```diff
@@ -0,0 +1,34 @@
+module "appservice_fims" {
+  source = "github.com/pagopa/terraform-azurerm-v3//app_service?ref=v7.67.1"
+
+  plan_type = "internal"
+  plan_name = "${var.product}-oidc-provider-plan"
+  sku_name  = "B3"
+
+  name                = "${var.product}-oidc-provider-app"
+  resource_group_name = var.resource_group_name
+  location            = var.location
+
+  always_on        = true
+  node_version     = "18-lts"
```

Comment:
if is available, we can switch to `20-lts`

Reply:
it is, I've updated the PR

Reply:
It does, I've updated it